src/pages/ja/core-concepts/astro-components.md

```diff
@@ -470,9 +491,21 @@ Astroは、これらのJavaScriptクライアントサイドインポートを
 </script>
 ```
 
+## HTMLコンポーネント
+
+Astroは`.html`ファイルをコンポーネントとしてインポートして使用したり、これらのファイルをページとして`src/pages`のサブディレクトリに設置することをサポートしています。フレームワークなしで構築された既存のサイトからコードを再利用したい場合や、コンポーネントに動的な機能がないことを確認したい場合はHTMLコンポーネントを利用するといいでしょう。
+
+HTMLコンポーネントは有効なHTMLしか含むことができず、そのためAstroコンポーネントの主要機能が制限されます
+- frontmatterやサーバーサイドのインポート、動的な記法をサポートしません
+- すべての`<script>`タグはバンドルされずに残され、`is:inline`を持つ場合と同じように扱われます。
+- [`public/`フォルダにあるアセット](/en/guides/images/#public)のみを参照できます。
+
+:::note
+HTMLコンポーネント内の[`<slot />`要素](/en/core-concepts/astro-components/#slots)はAstroコンポーネントと同様に機能します。[HTML Web Component のSlot](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot) 要素を使うためには,  `<slot>` 要素に`is:inline`を追加します。
```

Comment:
And we can update both our internal link and the MDN link here for the Japanese versions!

```suggestion
HTMLコンポーネント内の[`<slot />`要素](/ja/core-concepts/astro-components/#slots)はAstroコンポーネントと同様に機能します。[HTML Web Component のSlot](https://developer.mozilla.org/ja/docs/Web/HTML/Element/slot) 要素を使うためには,  `<slot>` 要素に`is:inline`を追加します。
```